src/routes/documents/[id]-[slug]/sidebar/Sections.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+  import type { Section } from "$lib/api/types";
+
+  import { ListOrdered16 } from "svelte-octicons";
+
+  import Empty from "@/lib/components/common/Empty.svelte";
+  import SidebarGroup from "@/lib/components/sidebar/SidebarGroup.svelte";
+  import SidebarItem from "@/lib/components/sidebar/SidebarItem.svelte";
+
+  export let sections: Section[];
+</script>
+
+<SidebarGroup>
+  <h2 slot="title">
+    <ListOrdered16 />
+    Sections
+  </h2>
```

Comment:
We can also use `SidebarItem` in the header position:
```suggestion
  <SidebarItem slot="title">
    <ListOrdered16 />
    Sections
  </SidebarItem>
```